src/components/PrimaryPanes/Sources.css

```diff
@@ -146,6 +146,8 @@
   user-select: none;
   box-sizing: border-box;
   height: 30px;
+  margin: 0;
+  padding: 0;
```

Comment:
Needed to add these because the container of the tabs is a `ul` now, instead of a `div`.